curriculum/challenges/english/10-coding-interview-prep/data-structures/remove-an-element-from-a-max-heap.md

```diff
@@ -120,10 +120,16 @@ assert(
       return false;
     }
     const removed = test.remove();
-    if(removed > max) return false
+    if (!vals.includes(removed)) return false;
+    if (removed > max) return false
```

Comment:
Since this file is being touched, the solution should be updated to remove 'var'